fxevent/console.go

```diff
@@ -63,24 +63,24 @@ func (l *ConsoleLogger) LogEvent(event Event) {
 		if e.Err != nil {
 			l.logf("ERROR\tFailed to supply %v: %v", e.TypeName, e.Err)
 		} else {
-			l.logf("SUPPLY\t%v", e.TypeName)
+			l.logf("SUPPLY\t%v from Module \"%v\"", e.TypeName, e.ModuleName)
```

Comment:
nit: use `%q` to let the system escape the module name.

also, let's just use "module" instead of "Module" since the concept of modules has already been introduced by this point. (there's some tech writing advice I read elsewhere that went along the lines of not capitalizing reusable concepts or components if they're not proper nouns.)

```suggestion
			l.logf("SUPPLY\t%v from module %q", e.TypeName, e.ModuleName)
```


same suggestion for the other log lines